docs/whats-new/release-notes/v2_15_0.md

```diff
@@ -0,0 +1,150 @@
+
+#### Zowe CLI Imperative Framework
+
+- Added multiple APIs to the `ProfileInfo` class to help manage schemas between client applications. ([#2012](https://github.com/zowe/zowe-cli/issues/2012))
+
+- Enhancement: Hid the progress bar if `CI` environment variable is set, or if `FORCE_COLOR` environment variable is set to `0`. ([#1845](https://github.com/zowe/zowe-cli/issues/1845))
+
+#### CICS Plug-in for Zowe CLI
+
+#### DB2 Plug-in for Zowe CLI
+
+#### MQ Plug-in for Zowe CLI
+
+#### IMS Plug-in for Zowe CLI
+
+- Added notices announcing the deprecation of the IMS Plug-in for Zowe CLI. ([#66](https://github.com/zowe/zowe-cli-ims-plugin/pull/66))
+
+#### z/OS FTP Plug-in for Zowe CLI
+
+- Simplified the preparation for JCL system tests.
+- Updated the version of zos-node-accessor to 1.0.16.
+
+### Zowe Explorer
+
+#### Zowe Explorer (Core)
+
+- See the [Zowe Explorer](https://github.com/zowe/vscode-extension-for-zowe/blob/main/packages/zowe-explorer/CHANGELOG.md) changelog for updates included in this release.
+
+#### Zowe Explorer API
+
+- See the [Zowe Explorer API](https://github.com/zowe/vscode-extension-for-zowe/blob/main/packages/zowe-explorer-api/CHANGELOG.md) changelog for updates included in this release.
+
+#### Zowe Explorer FTP Extension
+
+- See the [Zowe Explorer FTP Extension](https://github.com/zowe/vscode-extension-for-zowe/blob/main/packages/zowe-explorer-ftp-extension/CHANGELOG.md) changelog for updates included in this release.
+
+#### Zowe Explorer ESLint Plug-in
+
+- See the [Zowe Explorer ESLint Plug-in](https://github.com/zowe/vscode-extension-for-zowe/blob/main/packages/eslint-plugin-zowe-explorer/CHANGELOG.md) changelog for updates included in this release.
+
+## Bug fixes
+
+Zowe Version 2.15.0 contains the bug fixes that are described in the following topics.
+
+### Zowe installation and packaging
+ 
+### Zowe Application Framework
+
+#### Zlux Server Framework
+
+### Zowe API Mediation Layer
+
+ -Displaying the base path in API Catalog has been fixed. ([#3297](https://github.com/zowe/api-layer/issues/3297))
+- Disabling EhCache using `apiml.caching.enabled=false` has been fixed. ([#3280](https://github.com/zowe/api-layer/issues/3280))
+
+### Zowe CLI
+
+#### Zowe CLI (Core)
+
+- Fixed race condition in `config convert-profiles` command that may fail to delete secure values for old profiles.
+
+- Resolved technical currency by updating `socks` transitive dependency.
+
+- Updated zos-files copy dsclp system tests to include large mock files.
+
+#### Zowe CLI Imperative Framework
+
+- Fixed race condition in `config convert-profiles` command that may fail to delete secure values for old profiles.
+
+- Resolved issue in `ProfileInfo` where schema comparisons fail, specifically when comparing the cached schema against a command-based schema during registration.
```

Comment:
@adam-wolfe - What PR/issue link can we add to this item?

Reply:
https://github.com/zowe/zowe-cli/pull/2045